packages/status-js/src/client/activityCenter.ts

```diff
@@ -0,0 +1,143 @@
+// todo?: rename to notifications (center?), inbox, or keep same as other platforms
```

Comment:
This and other topics also raised at https://discord.com/channels/624205794384281629/1012712530780622870/1013748671793348678.